application.properties

```diff
@@ -0,0 +1,6 @@
+#Setting the count of user name and password.
+user_name=
+password_n=
+#Set the web Page URL
+url=
```

Comment:
Write some sentence to remember that here we need put values E.g. "url=Put url"

Reply:
Please dear rail this is a step that a considerate not relevant but in the next review a going implement that.